sbol_utilities/conversion.py

```diff
@@ -407,7 +423,7 @@ def command_line_converter(args_dict: Dict[str, Any]):
         raise ValueError(f'Unknown file type {input_file_type}; should have been caught earlier')
 
     # Convert output from SBOL3
-    logging.info('Writing output file '+output_file)
+    if doc3: logging.info('Writing output file '+output_file)
```

Comment:
Remove this if, and the one below, since we shouldn't ever have a None returned.